android/kotlin-central-login-oidc/app/src/main/java/com/example/app/Config.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2024. PingIdentity. All rights reserved.
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+
+package com.example.app
+
+import kotlinx.serialization.Serializable
+import org.forgerock.android.auth.FROptions
+import org.forgerock.android.auth.FROptionsBuilder
+
+@Serializable
+data class PingConfig(
```

Comment:
We should remove the information related to our environment here and add instructions to add their own.